packages/fhir-converter/src/templates/cda/Resources/Condition.hbs

```diff
@@ -12,10 +12,12 @@
         "clinicalStatus":
         [ 
             {{#each (toArray conditionEntry.entryRelationship)}}
-                {{#if (eq this.observation.templateId.root '2.16.840.1.113883.10.20.22.4.5')}} 
-                    {{>DataType/CodeableConcept.hbs code=this.observation.value}},
-                {{/if}}
-          {{/each}}
+                {{#each (toArray this.observation.templateId)}}
+                    {{#if (eq this.root '2.16.840.1.113883.10.20.22.4.6')}} 
```

Comment:
note, the `2.16.840.1.113883.10.20.22.4.5` had a one-digit typo, never evaluating to true